ui/notify/src/ctrl.ts

```diff
@@ -29,7 +29,8 @@ export default function makeCtrl(opts: NotifyOpts, redraw: Redraw): Ctrl {
 
   function attention() {
     const id = data?.pager.currentPageResults.find(n => !n.read)?.content.user?.id;
-    if (!lichess.quietMode || id == 'lichess') lichess.sound.playOnce('newPM');
+    const playBell = document.body.getAttribute('data-play-bell') === 'true';
```

Comment:
should be:
`    const playBell = lichess.storage.get('playBellSound') !== 'false';`